Build/Scripts/runTests.sh

```diff
@@ -308,10 +310,14 @@ PHP_XDEBUG_ON=0
 PHP_XDEBUG_PORT=9003
 EXTRA_TEST_OPTIONS=""
 PHPUNIT_RANDOM=""
+# CGLCHECK_DRY_RUN is a more generic dry-run switch not limited to CGL
 CGLCHECK_DRY_RUN=0
 DATABASE_DRIVER=""
 CONTAINER_BIN=""
 COMPOSER_ROOT_VERSION="3.0.x-dev"
+NODE_VERSION=18
```

Comment:
Could we also use a newer version, e.g. 20 or 22?

Reply:
The package.json doesn‘t allow any higher version, so that‘s a different story.

Reply:
Ack.